sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -332,6 +332,11 @@ def test_upretty_modifiers():
     assert upretty( Symbol('x__dot') ) == u('x__dot')
 
 
+def test_pretty_Cycle():
+    from sympy.combinatorics.permutations import Cycle
+    assert pretty(Cycle(1, 2)) == '([1, 2],)'
```

Comment:
> It's been a while, but I think this should appear as (1, 2)

Better to print as `(1 2)` with no comma in between? [Wolfram](https://www.wolframalpha.com/input/?i=perm+%281+2+3+4%29^3%281+2+3%29^-1&lk=3) also prints as this.